testcafe/tests/pages/photos-model.js

```diff
@@ -48,21 +36,48 @@ export default class Page {
     this.modalDelete = Selector('[class*="c-modal"]').find('div')
     this.modalDeleteBtnDelete = this.modalDelete.find('button').nth(2) //REMOVE
 
-    this.allPhotos = Selector('[class*="pho-photo"]').find(
-      '[class*="pho-photo-item"]'
+    this.allPhotosWrapper = this.photoSection.find('[class^="pho-photo"]')
+    this.allPhotos = Selector('img[class*="pho-photo-item"]')
+
+    // Photo fullscreen
+    this.photoFull = Selector('[class*="pho-viewer-imageviewer"]').find('img')
+    this.photoNavNext = Selector('[class*="pho-viewer-nav--next"]')
+    this.photoNavNextBtn = this.photoNavNext.find(
+      '[class*="pho-viewer-nav-arrow"]'
+    )
+    this.photoNavPrevious = Selector('[class*="pho-viewer-nav--previous"]')
+    this.photoNavPreviousBtn = this.photoNavPrevious.find(
+      '[class*="pho-viewer-nav-arrow"]'
     )
+    this.photoBtnClose = Selector('[class*="pho-viewer-toolbar-close"]').find(
+      '[class*="c-btn"]'
+    )
+  }
+
+  async initPhotoPage() {
+    await this.waitForLoading()
+    t.ctx.allPhotosStartCount = await this.getPhotosCount('Before')
```

Comment:
C'est quoi ce `ctx` ?

Reply:
un contexte testcafe qui permet de partager une variable entre les hooks (beforeEach) et les tests https://devexpress.github.io/testcafe/documentation/test-api/test-code-structure.html#sharing-variables-between-test-hooks-and-test-code

Reply:
C'est parce que j'ai juste modifier son initialisation, donc ses autres utilisations n'apparaissent pas dans la PR, mais je m'en sert notament pour ouvrir la dernière photos de la liste 
` await page.openPhotoFullscreen(t.ctx.allPhotosStartCount - 1)` (https://github.com/cozy/cozy-drive/blob/testcafe/testSelectors/testcafe/tests/photos_crud.js#L53), ou pour vérifier le nombre de photos attendues après un upload/delete :
```
await t
      .expect(allPhotosEndCount)
.eql(t.ctx.allPhotosStartCount + numOfFiles)
```  (https://github.com/cozy/cozy-drive/blob/testcafe/testSelectors/testcafe/tests/pages/photos-model.js#L105-L107)

Reply:
Ok mais du coup, est-ce qu'on peut attendre la prochaine PR pour introduire tout le changement d'un coup, plutôt que d'en faire un bout ici et un bout dans une autre PR ? 
Ca permet de mieux suivre les changements.

Reply:
Et du coup on m'indique que l'ancien init est dans photos_crud, tout est ok donc.